### PR TITLE
docs(Http): add type annotations to clarify API

### PR DESCRIPTION
**Pull request by gjungb**
IMHO this tiny example is easier to read when some type annotations are added and the parameter names are more concise.

### CODE REVIEW

**Comment by PascalPrecht** [comment]
@gjungb the change looks good, but could you please run `gulp check-format` and update this PR? 

Some builds are still failing due to invalid formatting. Thanks!

File: modules/examples/src/http/http_comp.ts [hunk 1]
```diff
@@ -14,8 +14,10 @@ import {Http} from 'angular2/http';
   `
 })
 export class HttpCmp {
-  people: Object;
+  people: Array<Object>;
```
Comment (vsavkin) [inline modules/examples/src/http/http_comp.ts]: can you use the `Object[]` syntax instead of `Array<Object>`

**Comment by vicb** [comment]
Could you please:
- integrate the feedback,
- squash your commits,
- follow the conventions for commit messages

**Comment by mhevery** [comment]
I have fixed the feedback and squashed. Merging...

**Comment by gjungb** [comment]
@mhevery thx for merging that!

**Pull request closed without merging**